src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int maxBallNum = 100;
+
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(maxBallNum + 1);
+        String randomColor = new ColorSupplier().getRandomColor();
```

Comment:
create global Random and ColorSupplier fields and use them here, so you won't need to create their instancec each time you call this method